docs/metrics/customizing-the-sdk/Program.cs

```diff
@@ -16,7 +16,12 @@ public class Program
     public static void Main()
     {
         using var meterProvider = Sdk.CreateMeterProviderBuilder()
-            .ConfigureResource(res => res.AddService("example-service"))
+            .ConfigureResource(r => r.AddAttributes(new List<KeyValuePair<string, object>>
```

Comment:
Use `resource` as the parameter name instead of `r`. We have started to use more readable user-friendly parameter names in our example: https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/docs/metrics/getting-started-aspnetcore/Program.cs#L11

We do need to update all examples to match this.